src/operator/contrib/count_sketch.cu

```diff
@@ -93,6 +93,9 @@ __global__ void sketch_backward_kernel(const int nthreads,
   // only calculate gradient regarding x
   // can also calculate gradient regarding s if needed
   const int index    = blockIdx.x * blockDim.x + threadIdx.x;
+  if (index >= nthreads) {
```

Comment:
That test was breaking the CI for a looong time, thanks for the fix :-). Could you search for the issues about it and close them once this is merged?

Reply:
This was a 'tack on' commit that was also in an already merged PR https://github.com/apache/incubator-mxnet/pull/20876, so it's part of master already. Per your suggestion here, I've now closed the open issue mentioning test_countsketch flakyness: https://github.com/apache/incubator-mxnet/issues/10988